teams/account-management/README.md

```diff
@@ -1,5 +1,54 @@
 # Account Management
 
+## Setup your local dev environment
+
+To get started on local development and testing:
+
+1. **Get your `buffer-dev` environment setup**
+  → https://github.com/bufferapp/buffer-dev
+
+2. **Install the latest version of `yarn`**
+  → [Installing Yarn](https://yarnpkg.com/en/docs/install)
+
+3. **Make sure you have node with version <= 9 (Node v10 is not compatible)**
+    ```
+    $ node -v
+    ```
+
+4. **Install Packages and Bootstrap**
```

Comment:
@hharnisc - is this true? I don't think buffer-login uses YARN right?

Reply:
Login uses `nvm` rather than `yarn`

Reply:
Do you mean `npm`? 😁